src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -1593,3 +1611,21 @@ private void disposeDebugging(DisposeTypes type)
 
     }
 }
+
+#if NUI_PROPERTY_TEST
+namespace Tizen.NUI
+{
+    internal enum InternalGetSetValueType
```

Comment:
I think we'd better make a lot of variation of `Interop.Actor.InternalSetValues` instead of make some new duplicated enum.

something like...

`Interop.Actor.InternalSetValuesVector2(SwigCPtr, propertyIndex, some_vector_2.SwigCPtr)`
`Interop.Actor.InternalSetValuesVector2(SwigCPtr, propertyIndex, some_vector_2.X, some_vector_2.Y)`
`Interop.Actor.InternalSetValuesPropertyMap(SwigCPtr, propertyIndex, some_property_map.SwigCPtr)`

Reply:
Thank you very much for your review.
I have changed in other patches following your comment.
and I close this "draft" PR now.
thank you again.